devel/query.lua

```diff
@@ -214,6 +214,21 @@ function getSelectionData()
     elseif args.job then
         debugf(0,"job selection")
         selection = dfhack.gui.getSelectedJob()
+        if selection == nil and df.global.cursor.x ~= -30000 then
```

Comment:
My teleport script is probably out of date then

Reply:
it will still be true sometimes. It looks like DF still initializes the cursor to -30000 to mark it as invalid, but then it continues to update it as the screen shifts, so you end up with a value that is near, but not exactly equal to, -30000